src/applications/my-education-benefits/config/form.js

```diff
@@ -337,11 +341,16 @@ function renderContactMethodFollowUp(formData, cond) {
   return res;
 }
 
+function transform(metaData, form) {
```

Comment:
This transform function is called presubmission and reads the form data and turns it into expected JSON.